eng/common/scripts/TypeSpec-Project-Generate.ps1

```diff
@@ -14,6 +14,80 @@ $ErrorActionPreference = "Stop"
 . $PSScriptRoot/common.ps1
 Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
 
+function GetNpmPackageVersion([string]$packagePath) {
+    $packagePath = Resolve-Path $packagePath
+    $packageFolder = Split-Path -Parent $packagePath
+    $packageName = Split-Path -Leaf $packagePath
+    Push-Location $packageFolder
+    try {
+        $packageVersion = node -p -e "require('./$packageName').version"
+        return $packageVersion
+    }
+    finally {
+        Pop-Location
+    }
+}
+
+function GetEmitterDependencyVersion([string]$packagePath) {
+    $matchString = "`"$emitterName`": `"(.*?)`""
```

Comment:
`$emitterName` is not set anywhere or passed in here.

Reply:
It is set globally in Line 121.